### ISSUE TITLE
Admin Dashboard: Retrieve Project Tokens should be visible when the vesting period ends

### ISSUE THREAD

**Issue by BartuCulha** [PrimeDAO/prime-launch-dapp#260]

**Comment by dkent600** [comment]
@BartuCulha I am thinking you may be confusing claiming and retrieving.  The admin retrieve button should give back all the project tokens that have not been purchased.  The amount you have purchased, locked or not, is not included in that amount.

The admin UI should be telling you how many tokens that would be.  What did it tell you?  According to your repro steps, it should say "0" but in fact, in that case, the retrieve section should not even be visible.  Sounds like that may be the true bug here.  Has nothing to do with vesting period.

**Comment by dkent600** [comment]
@BartuCulha Is this not a dup of https://github.com/PrimeDAO/prime-launch-dapp/issues/273 ?

**Comment by BartuCulha** [comment]
No, this one is about the button being visible in the dashboard. I have created another issue which is  #273 which is about displaying the  correct amount of retrievable tokens
